dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -281,6 +302,27 @@ def setLevels(self, mn, mx):
     def empty_function(self, *args):
         pass
 
+    def _configurationButtonClicked(self):
+        widget = ColormapPopup(parent=self.scene().views()[0])
```

Comment:
The popup widget needs a parent since there is no reference kept to it in Python.